.github/workflows/continuous-integration.yml

```diff
@@ -29,6 +29,11 @@ env:
   # email for account verification
   EMAIL_HOST_USER: ${{ secrets.EMAIL_HOST_USER }}
   EMAIL_HOST_PASSWORD: ${{ secrets.EMAIL_HOST_PASSWORD }}
+  # review drive account information
```

Comment:
Is this necessary to include in CI? (same for app.yaml.template, master-to-app-engine.yaml)

Reply:
I think it would be necessary if you were to run the script from prod given that it seems to take env information from github secrets on prod instead of the .env (I don't think prod has an .env) so I think it's probably necessary.